proxy/src/main.rs

```diff
@@ -20,13 +29,10 @@ async fn main() -> Result<()> {
         .without_time()
         .init();
 
-    // Load config file
-    let config = {
-        let config_path = std::env::args().nth(1).unwrap_or("config.toml".to_string());
+    let args: Arguments = argh::from_env();
```

Comment:
I'd keep comments like `// Load command arguments` and `// Load proxy config` because then the main function is readable even for people that doesn't know these specific libraries or implementation internals. After all this proxy is the prime example for how to use `imap-flow`.

Reply:
Okay! Let me reintroduce it.